dev/TeachingTip/TeachingTip_rs1_themeresources.xaml

```diff
@@ -7,19 +7,19 @@
         <ResourceDictionary x:Key="Default">
             <StaticResource x:Key="TeachingTipBorderBrush" ResourceKey="SurfaceStrokeColorDefaultBrush"/>
             <SolidColorBrush x:Key="TeachingTipTopHighlightBrush" Color="#0DFFFFFF"/>
-            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SystemControlPageBackgroundChromeLowBrush"/>
+            <StaticResource x:Key="TeachingTipTransientBackground" ResourceKey="SolidBackgroundFillColorTertiaryBrush"/>
```

Comment:
Should the Transient Background be solid, and not an Acrylic Brush?

Reply:
@mdtauk it is Acrylic since rs2 :) before rs2 - it's solid

Reply:
> @mdtauk it is Acrylic since rs2 :) before rs2 - it's solid
> check _rs2_themeresources

Ah I did not notice this was an RS1 only change.